### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CoderDS**
Vagrantfile fix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Octopeer
 
 ## Getting Started
+0. Change '/home/aaronang/Code' to the directory of your repository on your host machine.
```
Comment (aaronang) [inline README.md]: I would rather see this as second step. In  addition, I would use backticks to indicate the path. Please also indicate in which file this should be changed.

**Comment by CoderDS** [comment]
I prefer seeing this as an intermediate step between the bash commands.
Backticks are used and the indication to the file is present now.

**Comment by CoderDS** [comment]
Added ../ to the requirements.txt as otherwise the file won't be found as it is one directory upwards.

**Comment by CoderDS** [comment]
Added ../ to the manage.py as otherwise the file won't be found as it is one directory upwards.

**Comment by CoderDS** [comment]
Added the right directory in which the octopeer directory lives inside vagrant as some complaints where made about this being unclear from the readme.

File: README.md [hunk 2]
```diff
@@ -14,23 +14,29 @@ Afterwards, let Vagrant set up a fully running development box and SSH into it.
    ```bash
    host$ git clone git@github.com:theSorcerers/octopeer.git
    host$ cd octopeer
+    ```
+    In the Vagrantfile change: 
+    config.vm.synced_folder "/home/aaronang/Code", "/home/vagrant/Code"
```
Comment (aaronang) [inline README.md]: Use single backticks.

File: README.md [hunk 3]
```diff
@@ -14,23 +14,29 @@ Afterwards, let Vagrant set up a fully running development box and SSH into it.
...
+    to
+    config.vm.synced_folder "The absolute path to your just cloned repository", "/home/vagrant/Code"
```
Comment (aaronang) [inline README.md]: Use single backticks.
Also, I think it is better to do something like: `"/absolute/path/to/repository"`.

File: README.md [hunk 4]
```diff
@@ -14,23 +14,29 @@ Afterwards, let Vagrant set up a fully running development box and SSH into it.
...
+    ```bash
    host$ vagrant up
    host$ vagrant ssh
    ```
-1. Go to the project directory, and initialize and activate [virtualenv](https://virtualenv.pypa.io/en/latest/).
+1. Go to the project directory (The directory Code), and initialize and activate [virtualenv](https://virtualenv.pypa.io/en/latest/).
```
Comment (aaronang) [inline README.md]: No need for "The" to be capitalized.

**Comment by aaronang** [comment]
Some minor improvement suggestions. I will merge it after you have updated your submission.

**Comment by CoderDS** [comment]
Minor changes have been made.

File: README.md [hunk 5]
```diff
@@ -14,23 +14,29 @@ Afterwards, let Vagrant set up a fully running development box and SSH into it.
...
+1. Go to the project directory (the directory Code), and initialize and activate [virtualenv](https://virtualenv.pypa.io/en/latest/).
 Then, install all dependencies with [pip](https://pip.pypa.io/en/stable/).
 
    ```bash
-   vagrant$ cd octopeer
+   vagrant$ cd Code/octopeer
    vagrant$ virtualenv -p python3 env
    vagrant$ source env/bin/activate
-   vagrant$ pip install -r requirements.txt
+   vagrant$ pip install -r ../requirements.txt
```
Comment (aaronang) [inline README.md:33]: Shouldn't it just be `requirements.txt` without `../`?

File: README.md [hunk 6]
```diff
@@ -14,23 +14,29 @@ Afterwards, let Vagrant set up a fully running development box and SSH into it.
...
    ```
 
 1. You are ready to start hacking, fire up the server!
 
    ```bash
-   vagrant$ python manage.py runserver 0.0.0.0:8000
+   vagrant$ python ../manage.py runserver 0.0.0.0:8000
```
Comment (aaronang) [inline README.md:40]: Ditto

Comment (CoderDS) [inline README.md:40]: It is right:)

File: README.md (hunk 5 above)
Comment (CoderDS) [inline README.md:33]: No that shouldn't be.

Comment (CoderDS) [inline README.md:33]: Nop.

**Pull request merged**